### PR TITLE
Generate CSV for OLM

### PR DESCRIPTION
**Pull request by mrogers950**
Generated using `$ operator-sdk olm-catalog gen-csv --csv-version "0.1.0"`

### CODE REVIEW

**Comment by mrogers950** [comment]
/hold

Need to fill in a bunch of the fields.

File: deploy/olm-catalog/operator-source.yaml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+apiVersion: operators.coreos.com/v1
+kind: OperatorSource
+metadata:
+  name: file-integrity-operator
+  namespace: openshift-marketplace
+spec:
+  type: appregistry
+  endpoint: https://quay.io/cnr
```
Comment (JAORMX) [inline deploy/olm-catalog/operator-source.yaml]: might wanna change this.

Comment (mrogers950) [reply deploy/olm-catalog/operator-source.yaml]: According to https://github.com/operator-framework/operator-marketplace/blob/master/README.md this is the correct value

Comment (JAORMX) [reply deploy/olm-catalog/operator-source.yaml]: OK

**Pull request merged**